fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/controlTokens/AvatarTokens.kt

```diff
@@ -150,6 +150,24 @@ enum class AvatarSize {
     Size72
 }
 
+enum class SlicedAvatarSize {
```

Comment:
why do we need these? where are they used? are these fluent specific?

Reply:
removed this